### DISCUSSION TITLE
Update failure to 1.05

### DISCUSSION THREAD

**Discussion by Rolli1962** [akeeba/panopticon#353] [General]
Had just done the update to 1.05. 
Update got the error message (see attached PDF)
[pan-updatefailure-105.pdf](https://github.com/akeeba/panopticon/files/13522907/pan-updatefailure-105.pdf)
After this site seems to be on Version 1.05 - but variables for language are not working. My browser settings are to display German first
![pan-upd-105](https://github.com/akeeba/panopticon/assets/60449692/9d0544a2-99ba-43d4-8e13-f98c624a7ffc)

Maybe helpful for others - changed in my profile to English, than it worked.

**Comment by nikosdion** [comment]
Can you please try extracting the ZIP file for 1.0.5 over your current installation? Does that work with German?

**Reply by Rolli1962** [reply]
Have extracted the ZIP-File over. But no - do not work in German. Still the same.

**Reply by brianteeman** [reply]
The problem is that the german translation file only has 229 lines instead of 1695 lines

**Comment by nikosdion** [comment]
This issue is addressed in 1.0.6, using en-GB to fill in missing translations.
